sources/api/migration/migrations/v1.20.0/public-admin-container-v0-11-8/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "public-admin-container-v0-11-8"
+version = "0.1.0"
+authors = ["Kyle Sessions <kssessio@amazon.com>"]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+publish = false
+# Don't rebuild crate just because of changes to README.
+exclude = ["README.md"]
+
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+migration-helpers = { path = "../../../migration-helpers", version = "0.1.0"}
```

Comment:
Needs a newline at end of the file.

Reply:
Alright I have verified the new line in all the Cargo.toml files, thanks!